src/js/components/products/detail.tsx

```diff
@@ -75,130 +74,82 @@ const ProductDetail = (props: RouteComponentProps) => {
+        repoUrl={product.repo_url}
+        breadcrumb={[{ name: product.name }]}
+        sidebar={sidebarContent}
+      >
+        {!projects || !projects.fetched ? (
+          // Fetching projects from API
+          <Spinner />
+        ) : (
+          <>
+            <h2 className="slds-text-heading_medium slds-p-bottom_medium">
+              {projects.projects.length ? (
+                <>
+                  {i18n.t('Projects for')} {product.name}
+                </>
+              ) : (
+                <>
+                  {i18n.t('Create a Project for')} {product.name}
+                </>
+              )}
+            </h2>
+            <ProjectForm
+              product={product}
+              startOpen={!projects.projects.length}
+            />
+            {Boolean(projects.projects.length) && (
               <>
-                <h2 className="slds-text-heading_medium slds-p-bottom_medium">
-                  {projects.projects.length ? (
-                    <>
-                      {i18n.t('Projects for')} {product.name}
-                    </>
-                  ) : (
-                    <>
-                      {i18n.t('Create a Project for')} {product.name}
-                    </>
-                  )}
-                </h2>
-                <ProjectForm
-                  product={product}
-                  startOpen={!projects.projects.length}
-                />
-                {Boolean(projects.projects.length) && (
-                  <>
-                    <ul className="slds-has-dividers_bottom">
-                      {projects.projects.map(project => (
-                        <ProjectListItem
-                          key={project.id}
-                          project={project}
-                          product={product}
-                        />
-                      ))}
-                    </ul>
-                    {projects.next ? (
-                      <div className="slds-m-top_large">
-                        <Button
-                          label={
-                            fetchingProjects ? (
-                              <LabelWithSpinner
-                                label={i18n.t('Loading…')}
-                                variant="base"
-                                size="x-small"
-                              />
-                            ) : (
-                              i18n.t('Load More')
-                            )
-                          }
-                          onClick={fetchMoreProjects}
-                        />
-                      </div>
-                    ) : null}
-                  </>
-                )}
+                <ul className="slds-has-dividers_bottom">
+                  {projects.projects.map(project => (
+                    <ProjectListItem
+                      key={project.id}
+                      project={project}
+                      product={product}
+                    />
```

Comment:
At what point do we want to have list-components that we render, rather than having the enclosing `ul` just verbatim in the calling component? I don't think we need that here, but I want to make sure we have thoughts about when that's appropriate and when it's unnecessary.

Reply:
At this point this is the only list-item we're using, so I haven't thought a lot about best points of abstraction yet.